### DISCUSSION TITLE
Question about vertico-quick-insert & consult-grep

### DISCUSSION THREAD

**Discussion by uqix** [minad/vertico#455] [Q&A] [answered]
<img width="685" alt="image" src="https://github.com/minad/vertico/assets/10142314/2cf9c3e8-28ca-4844-bcd7-2150a25be608">
<img width="674" alt="image" src="https://github.com/minad/vertico/assets/10142314/050eaf83-dac3-4e37-abfc-71bc1a6d8d6a">

Now press `a`:

<img width="673" alt="image" src="https://github.com/minad/vertico/assets/10142314/6ab29cae-0d46-4bc6-b7ba-cef54263425b">

the result is kind of unhelpful.

**Comment by minad** [comment, marked as answer]
`vertico-insert` and `vertico-quick-insert` insert the currently selected candidates. This is indeed not helpful in this context, but it makes sense for `find-file`. You probably want to use `vertico-quick-exit` instead.

**Reply by uqix** [reply]
`vertico-quick-exit` works perfectly.
